src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -5,87 +5,172 @@
 
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
+import seedu.address.model.person.ModuleRoleContainsKeywordsPredicate;
 import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
     @Test
-    public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
+    public void equals() throws ParseException {
+        NameContainsKeywordsPredicate firstNamePredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
+        NameContainsKeywordsPredicate secondNamePredicate =
                 new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+        ModuleRoleContainsKeywordsPredicate firstModuleRolePredicate = new ModuleRoleContainsKeywordsPredicate(
+                ParserUtil.parseModuleRolePairs(Collections.singletonList("CS1010")));
+        ModuleRoleContainsKeywordsPredicate secondModuleRolePredicate = new ModuleRoleContainsKeywordsPredicate(
+                ParserUtil.parseModuleRolePairs(Collections.singletonList("CS2103T-prof")));
 
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
+        List<Predicate<Person>> firstNamePredicates = new ArrayList<>();
+        firstNamePredicates.add(firstNamePredicate);
+
+        List<Predicate<Person>> secondNamePredicates = new ArrayList<>();
+        secondNamePredicates.add(secondNamePredicate);
+
+        List<Predicate<Person>> firstModuleRolePredicates = new ArrayList<>();
+        firstModuleRolePredicates.add(firstModuleRolePredicate);
+
+        List<Predicate<Person>> secondModuleRolePredicates = new ArrayList<>();
+        secondModuleRolePredicates.add(secondModuleRolePredicate);
+
+        List<Predicate<Person>> firstNameAndModuleRolePredicates = new ArrayList<>();
+        firstNameAndModuleRolePredicates.add(firstNamePredicate);
+        firstNameAndModuleRolePredicates.add(firstModuleRolePredicate);
+
+        List<Predicate<Person>> secondNameAndModuleRolePredicates = new ArrayList<>();
+        secondNameAndModuleRolePredicates.add(secondNamePredicate);
+        secondNameAndModuleRolePredicates.add(secondModuleRolePredicate);
```

Comment:
can consider using `Arrays.asList()` to define the contents of the `List` inline